packages/specs/schema/test/integrations/discriminator.enum.integration.spec.ts

```
@@ -0,0 +1,1624 @@
+  Post,
+  Property,
+  Put,
+  Required,
+  Returns
+} from "../../src/index.js";
+
+export enum EventType {
+  PAGE_VIEW = "page_view",
+  ACTION = "action",
+  CLICK_ACTION = "click_action"
+}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)